stl/debugger/STL.natvis

```diff
@@ -337,6 +337,18 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
       </Expand>
   </Type>
 
+  <Type Name="std::unique_ptr&lt;wchar_t [0],*&gt;">
+    <AlternativeType Name="std::unique_ptr&lt;char [0],*&gt;"/>
+    <AlternativeType Name="std::unique_ptr&lt;char8_t [0],*&gt;"/>
+    <AlternativeType Name="std::unique_ptr&lt;char16_t [0],*&gt;"/>
+    <AlternativeType Name="std::unique_ptr&lt;char32_t [0],*&gt;"/>

```

Comment:
What about `unique_ptr<const char[]>` etc?

(Note that `vector<const char>` isn't a thing / shouldn't be a thing, but `unique_ptr<const T>` is common and good for safety.)

Reply:
I understand how `unique_ptr<const ...>` could be useful in general. But can you think of any case where one would want a `unique_ptr<const CHAR_TYPE[]>`. The places where I think a `unique_ptr<CHAR_TYPE[]>` would be useful are for making a temporary character buffer to, for example, call a Windows API. But I can't think of a case where you would want that to be `const`.

Reply:
I think it's good to add anyways; it's not necessarily a bad thing to add the `const` versions.

Reply:
Note that it's possible to construct a `unique_ptr<char[]>`, fill it with data, and then *convert* it to `unique_ptr<const char[]>` so nobody can alter the contents.